setup.py

```diff
@@ -77,7 +77,7 @@
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=['openpyxl==2.0.5'],
+    install_requires=['openpyxl==2.6.0'],
```

Comment:
libraries should generally never include versions of dependencies, and if they do they should be bounded with `~=`, `>`, `<`, etc. When a library uses `==` it forces all the downstream dependencies to use that version as well which can make package upgrades a headache.

I think removing the version completely is probably best for this library

Also note that HQ will be using 2.6.1 after https://github.com/dimagi/commcare-hq/pull/23504 so this would immediately cause a conflict if you tried add this to our dependencies

Reply:
> I think removing the version completely is probably best for this library

when i upgraded things failed so now i have made this compatible with '2.6.1'. Going to change this to 
`'openpyxl>=2.6.1'`